t/integration/test_canvas.py

```diff
@@ -28,6 +28,21 @@ def test_complex_chain(self, manager):
         assert res.get(timeout=TIMEOUT) == [64, 65, 66, 67]
 
     @flaky
+    def test_group_chord_group_chain(self, manager):
+        if not manager.app.conf.result_backend.startswith('redis'):
+            raise pytest.skip('Requires redis result backend.')
+        from redis import StrictRedis
+        redis_connection = StrictRedis()
+
```

Comment:
Perhaps it would be best if you cleared the Redis list, just in case the test is repeated:

```python
redis_connection.delete('redis-echo')
```